en/components/expansion_panel.md

```diff
@@ -34,19 +34,21 @@ We can now start using `igx-expansion-panel` in the markup. The expansion panel
         - `igx-expansion-panel-description` - stores the component description. Can be used to provide a short summary. Will always be displayed second in the headers.
```

Comment:
"Will always be displayed second in the headers" could be misleading as there could be no title at all. Can be something like after a title if title is set.